api/response/models.py

```diff
@@ -22,7 +22,7 @@ class Response(Base, Utility):
     question_id = Column(Integer, ForeignKey('questions.id'))
     rate = Column(Integer, nullable=True)
     check = Column(Boolean, nullable=True)
-    text_area = Column(String, nullable=True)
+    additional_comment = Column(String, nullable=True)
```

Comment:
Why changing this field to additional comment, in the question model, we have three field input, check, and rate. The text_area field is meant to take all input responses, while rate is for rating responses and check for missing_items, Instead of changing the text_area field to additional comment, you can always assign `additional comment` to the `text_area field` in your query implementation.  Other query are using that field which might make the app crash the moment you changed the name. Or what did you think?

Reply:
@cwizard2011, the reason was because I feel like it is not descriptive. For example, having the rate field as "integer_field". One does not know at first glance what that field does. It is a very quick fix and the app at the moment is not crashing on my side.

Reply:
alright, that is cool